network-utils/src/main/java/com/stormeye/network/CrossOriginConfig.java

```diff
@@ -1,9 +1,10 @@
 package com.stormeye.network;
 
 import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestMethod;
 
 /**
  * Marker Interface for CORS configuration
  */
-@CrossOrigin(origins = {"audit.stormeye2000.com", "store.stormeye2000.com"})
+@CrossOrigin(origins = {"audit.stormeye2000.com", "store.stormeye2000.com"}, methods = {RequestMethod.GET})
```

Comment:
Can these come from system properties? We don't want URLs hard coded here, if not we need to find another solution